src/LibraryManager.Vsix/ErrorList/ErrorList.cs

```diff
@@ -47,6 +47,16 @@ public bool HandleErrors(IEnumerable<ILibraryInstallationResult> results)
             return Errors.Count > 0;
         }
 
+        public void HandleError(IError error)
+        {
+            Errors.Add(new DisplayError(error));
```

Comment:
In general, we should probably try to figure out the line/column of the error, otherwise it will always be (0,0). The way the code currently does it in HandleErrors and AddLineAndColumn it pretty horrible though. We don't need it for the particular error that this fix is adding, but should clean-up this code in the next release.

In general, I am not a fan of how this class is implemented. We create and throw away instance of it all the time, It seems like there should be just one instance, or at least one per project. Also I haven't looked at how the errors get cleared, I am assuming it works somehow since this mostly follows what HandleErrors method does for multiple errors. Might be good to check though.

Reply:
Yes, I agree. I will open a bug for us to revisit the ErrorList class design.

Reply:
They were not getting cleared when closing the file. I have a fix for that on another PR once this one goes in.